### ISSUE TITLE
Add IRA-like calculations to economics module

### ISSUE THREAD

**Issue by malcolm-dsider** [NREL/GEOPHIRES-X#170]
Add a simple way to estimate the impact of IRA-like tax incentives on a geothermal project:
1) Investment Tax Credit - applied to reduce CAPEX?
2) Production Tax Credit - applied to reduce OPEX and thereby reduce LCOE?

Need to account for "adders" like 10% for local content, 10% for Energy Community, etc.
Also need to account for time limits (e.g., PTC expires 10 years after it is started

**Comment by softwareengineerprogrammer** [comment]
@malcolm-dsider is this related to and/or duplicative of https://github.com/NREL/GEOPHIRES-X/issues/169?title=Additional+economic+calculations?

**Comment by malcolm-dsider** [comment]
No. Related and perhaps executed at the same time, but different calculations.

**Comment by malcolm-dsider** [comment]
Completed via PR #177
